docker-compose.normalization.build.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3.7"
```

Comment:
it is still valuable to leverage docker-compose here to guarantee we get the same env variables as the rest of the core app. but ultimately we want the config to be separate because the normalization image is not run as part of compose up.

Reply:
keep in mind we will kill this normalization container at some point too and roll it into the destination, so this will hopefully be short-ish lived. but definitely around for several weeks.

Reply:
Since Normalization has the same lifecycle as the rest of core, it shouldn't be separate. I don't see `docker-compose.build.yaml` as something that tells us what to run but what to build for core. So it is ok if it is declared as a `service`.

Having every "Core" artifacts listed in the same place will save us a lot of headache or incomplete releases.

Reply:
Normalization does _not_ have the same lifecycle as the rest for core. All of core is run on `docker-compose up`. Normalization is _not_. Normalization is run when it is invoked from the worker.

Reply:
i think michel is suggesting adding it to the `docker-compose.build.yaml` file not `docker-compose.yaml`

Reply:
discussed offline and agreed to keep it separate. the issue with adding to the `docker-compose.build.yaml` file is that when we push to docker hub to release we rely on both `docker-compose.yaml` AND `docker-compose.build.yaml`. They stack on each other, so `docker-compose.yaml` is part of the build / release process, but it is also how the containers get run. Agreed for now to keep it separate. But we all think it is sad. 😭